shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/spi/ordered/OrderedSPIRegistry.java

```diff
@@ -77,18 +77,18 @@
      * @param comparator comparator
      * @return registered services
      */
-    @SuppressWarnings("unchecked")
     public static <K, V extends OrderedSPI<?>> Map<K, V> getRegisteredServices(final Collection<K> types, final Class<V> orderedSPIClass, final Comparator<Integer> comparator) {
```

Comment:
Is it better to adjust the param sequence too?
`orderedSPIClass` maybe the first param, `types` is second one.

Reply:
> Is it better to adjust the param sequence too?
> `orderedSPIClass` maybe the first param, `types` is second one.

@terrymanu Yes, it is better to understand. I will optimize it.